src/AutoRest.CSharp/Common/Generation/Writers/ModelWriter.cs

```diff
@@ -78,15 +81,33 @@ private void WriteObjectSchema(CodeWriter writer, ObjectType schema)
 
         protected virtual void WriteProperties(CodeWriter writer, ObjectType schema)
         {
+            var rawDataField = (schema as SerializableObjectType)?.RawDataField;
             foreach (var property in schema.Properties)
             {
+                if (property == rawDataField)
+                    continue;
+
                 WriteProperty(writer, property);
 
                 if (property.FlattenedProperty != null)
                     WriteProperty(writer, property.FlattenedProperty);
             }
         }
 
+        // TODO -- this is workaround
```

Comment:
we need to consolidate the way we treat FieldDeclaration and properties first. I will update the comment.
Here I say it is a workaround because this thing is now declared as a property (because this is the only way to include it into the ctor now without massive changes) but written as a field.